src/ground_truth_nsrts.py

```diff
@@ -2763,17 +2763,17 @@ def _get_predicate(base_pred_name: str,
         pred_name = f"{base_pred_name}-{type_names}"
         return pred_name_to_pred[pred_name]
 
-    # We start by creating reachable predicates for all possible type
-    # combinations. These predicates will be used as side predicates
-    # for navigateTo operators
+    agent_type = type_name_to_type["agent"]
+    agent_obj = Variable("?agent", agent_type)
+
+    # We start by creating reachable predicates for the agent and
+    # all possible other type combinations. These predicates will
```

Comment:
do you mean "types" instead of "type combinations"? you're no longer doing a combination here -- that's precisely what this change is removing!